tsconfig-common.json

```diff
@@ -26,6 +26,6 @@
 
         /* Completeness */
         "skipDefaultLibCheck": true, // it's safe to assume that built-in .d.ts files are correct
-        "skipLibCheck": false
+        "skipLibCheck": true, // there are conflicts between some of the globals in various third-party packages
```

Comment:
**question[change-required]:** Can you be more specific here? I think we should document which packages and which conflicts somewhere (either here or in a linked Jira ticket) so that we know when this check might be changed to `false` again.

Reply:
Good idea, will do.